helm/cluster-azure/templates/_externaldnsprivate_app_config.yaml

```diff
@@ -0,0 +1,22 @@
+{{/* Azure-specific external-dns-private Helm values*/}}
+{{/* https://github.com/giantswarm/external-dns-app/blob/main/helm/external-dns-app/values.yaml*/}}
+{{- define "azureExternalDnsPrivateHelmValues" }}
+provider: azure-private-dns
+clusterID: {{ include "resource.default.name" $ }}
+crd:
+  install: false
+domainFilters:
+  - "{{ include "resource.default.name" $ }}.{{ .Values.global.connectivity.baseDomain }}"
+txtOwnerId: giantswarm-io-external-dns
+txtPrefix: {{ include "resource.default.name" $ }}
+sources:
+  - service
```

Comment:
So external-dns doesn't need to watch for ingress, is that fine?

Reply:
To be honest, I am not fully sure since we have a bug in dns-operator-azure. We don't create wildcard to private DNS zones. But it is another discussion. I moved the config as it is here. I am fixing the overwriting issue now.